.github/workflows/ci_docker.yml

```diff
@@ -0,0 +1,88 @@
+name: CI docker
+
+on:
+  push:
+    branches:
+      - develop
+      - master
+      - GPL-769-cherrypicked-samples-view-revisited
+    paths-ignore:
+      - "README.md"
+    tags:
+      - uat-*
+      - release-*
+      - v*
+env:
+  IMAGE_NAME: ${{ github.repository }}/${{ github.event.repository.name }}
+  EVENT_WAREHOUSE_IMAGE: docker.pkg.github.com/sanger/event_warehouse/event_warehouse:gpl769_test_cp_view
```

Comment:
This needs to be updated to ```docker.pkg.github.com/sanger/event_warehouse/event_warehouse:master``` after the dependent pull request is merged.

Reply:
The dependent pull request is: https://github.com/sanger/event_warehouse/pull/135

Reply:
Changed to master